api/provider/aws/README.md

```diff
@@ -242,12 +245,18 @@ func (e *EntityProvider) readResourceB(args) (*aws.ResourceB, error) {
 		return nil, err
 	}
 
+    // assuming `Describe()` returns `(ECSOutput, err)`
+    // and `ECSOutput.ResourceBs` is `[]*ResourceB`
 	output, err := e.AWS.ECS.Describe(input)
 	if err != nil {
 		return nil, err
```

Comment:
`return nil, errors.Newf(errors.<Entity>DoesNotExist, "<Entity> %s does not exist", <entity>ID)`

Reply:
✅